integrations/docker/run.sh

```diff
@@ -35,7 +35,7 @@ ORG=${DOCKER_RUN_ORG:-connectedhomeip}
 IMAGE=${DOCKER_RUN_IMAGE:-$(basename "$here")}
 
 # version
-VERSION=${DOCKER_RUN_VERSION:-$(cat "$here/version")} ||
+VERSION=${DOCKER_RUN_VERSION:-latest} ||
```

Comment:
we haven't been keeping latest up to date...

Reply:
Using `$(cat "$here/version")` with cause the cirque build to fail. Is it okay to ignore it?

Reply:
We should probably just use a specific version like `0.4.11`, like GitHub actions does. That avoids the circular dependency when updating the dockerfile.

Reply:
* can't ignore cirque builds
* can't use a specific version (because version is shared, via symlinks)

For this PR to land, **_all_** of the images need to be pushed first.

This should probably be looked at again.  When I wrote all this, it was expedient to bump all versions all the time because chip-build was the image that changed most often and all the children needed the latest chip-build.

Today, we can version chip-build independently and let children update as necessary.

Changes needed:
* images/*/version files to be un-symlinked
* Dockerfile FROM directives to use specific versions
* integrations/docker/images/build-all.sh to be reworked with the above in mind

Reply:
I've unlinked the version file in build-cirque and fix it to v0.4.11 as a temporary fix.